packages/cli/src/commands/experimental/util.js

```diff
@@ -44,17 +44,8 @@ export const printTaskEpilogue = (command, description, topicId) => {
   )
 }
 
-export const serverFileExists = () => {
-  const serverFilePath = path.join(
-    getPaths().api.src,
-    `server.${isTypeScriptProject() ? 'ts' : 'js'}`
-  )
-
-  return fs.existsSync(serverFilePath)
-}
-
 export const isServerFileSetup = () => {
-  if (!serverFileExists) {
+  if (!serverFileExists()) {
```

Comment:
Seems to have been a bug here where we were just checking the truthiness of a function